docs/modules/migration/partials/migration-guide-24.1.adoc

```diff
@@ -689,3 +689,81 @@ The TypeScript API to access CodeTypes and Codes has been streamlined and simpli
 ** `CodeType.optGet`: Has been deleted as `CodeType.get` can be used now (see above).
 ** `CodeType.codes`: It is no longer a property but a function instead which by default returns all `Codes` of the `CodeType` recursively. It has a parameter which allows to only return the root `Codes`.
 ** `CodeType.getCodes`: has been removed. Use the new function `CodeType.codes()` instead (see above).
+
+== Tree: Improved auto-check behavior when checking nodes
+
+=== New behavior explained
+
+When the `autoCheckChildren` property of the Tree or TreeBox widget is set to `true` (e.g. by overriding
+`getConfiguredAutoCheckChildNodes()`) the automatic check behavior has changed.
+
+Assuming, the following Tree structure is given:
```

Comment:
I don't think there is a comma necessary. Also, I think tree should be preferred in this sentence instead of Tree.